docs/config/config.md

```diff
@@ -4,144 +4,149 @@ Config is a datastructure which holds all of the data for a Claudie microservice
+  | ---------- | --------------------- | ------------------------------------------------------- |
+  | Name       | string                | Name of the node pool                                   |
+  | Region     | string                | Region of the nodes                                     |
+  | ServerType | string                | Machine type of the nodes                               |
+  | Image      | string                | OS image of the nodes                                   |
+  | DiskSize   | int32                 | Disk size of the nodes                                  |
+  | Zone       | string                | Zone for the nodes                                      |
+  | Count      | int32                 | Count of the nodes                                      |
+  | Nodes      | [][Node](#node)       | Slice of Nodes                                          |
+  | Provider   | [Provider](#provider) | Provider of the nodepools                               |
+  | IsControl  | bool                  | Flag to differentiate between control and compute nodes |
 
 ## Node
 Node represent a single node from the nodepool
 
-  | Name | Type | Description |
-  |------|------|-------------|
-  | Name | string | Name of the node |
-  | Private | string | Private IP of the node in the VPN |
-  | Public | string | Public IP of the node |
-  | NodeType | [NodeType](#nodetype) | Type of the node |
+  | Name     | Type                  | Description                       |
+  | -------- | --------------------- | --------------------------------- |
+  | Name     | string                | Name of the node                  |
+  | Private  | string                | Private IP of the node in the VPN |
+  | Public   | string                | Public IP of the node             |
+  | NodeType | [NodeType](#nodetype) | Type of the node                  |
   
 ## Provider
 Provider represent a single provider from manifest
 
-  | Name | Type | Description |
-  |------|------|-------------|
-  | SpecName | string | Provider name |
-  | Credentials | string | Credentials of the provider |
-  | GcpProject | string | GCP project (only required when using GCP as DNS provider) |
-  | CloudProviderName | string | Cloud provider name. e.g. `gcp` and `hetzner`
+  | Name              | Type   | Description                                                |
+  | ----------------- | ------ | ---------------------------------------------------------- |
+  | SpecName          | string | Provider name                                              |
+  | Credentials       | string | Credentials of the provider                                |
+  | GcpProject        | string | GCP project (only required when using GCP as DNS provider) |
+  | CloudProviderName | string | Cloud provider name. e.g. `gcp`, `hetzner` or `oci`        |
+  | UserOcid          | string | OCID of the user                                           |
```

Comment:
This might be a good idea. I already thought about appending prefixes to variables, like `oci_`, `gcp_`, and `general_`, but I didn't want to "pollute" the PR more